features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
This is certainly a question of taste. But not only! Here are some functional advantages of my style over yours:

1. it better distinguishes a block of arguments from an indented block on the next line.
  ```coffeescript
  email.send(
    new Smtp().instance(),
    new Proxy().connect('gateway'))
  ```
  Does it connect the Proxy instance before or after sending the email? This looks too similar to 

  ```coffeescript
  email.send ->
    new Smtp().instance()
    new Proxy().connect('gateway')
  ```
  There is only tiny amount of hardly visible syntax boilerplate that distinguishes these two forms.

  This style makes it abundantly clear what's going on: 

  ```coffeescript
  email.send new Smtp().instance,
             new Proxy()
  ```

1. it allows to provide inline methods without requiring parentheses
  ```coffeescript
  user.load '@charlierudolph', 
            (done) ->
              done 'really done now'
  ```
  vs
  ```coffeescript
  user.load(
    '@charlierudolph', 
    (done) ->
      done 'really done now')
  ```

1. it doesn't require parentheses
1. it needs less vertical lines
  ```coffeescript
  foo bar,
      baz
  ```

  vs
  ```coffeescript
  foo(
    bar,
    baz)
  ```
  or even worse, if the parentheses are balanced:
  ```coffeescript
  foo(
    bar,
    baz
  )
  ```

1. it better uses the available horizontal space (see above example)

1. it integrates seamlessly and naturally with method calls that have only a single argument

  ```coffeescript
  foo 'one'

  bar 'one',
      'two'
  ```
  vs

  ```coffeescript
    foo 'one'

    bar(
      'one',
      'two')
  ```

1. it better visualizes the logical structure of the code. The arguments belong directly to the method call, hence they are behind it.